okhttp-clients/src/main/java/com/palantir/conjure/java/okhttp/ConcurrencyLimiters.java

```diff
@@ -204,42 +204,41 @@ public void onDropped() {}
     }
 
     final class DefaultConcurrencyLimiter implements ConcurrencyLimiter {
-
         @GuardedBy("this")
-        private final ThreadWorkQueue<SettableFuture<Limiter.Listener>> waitingRequests = new ThreadWorkQueue<>();
+        private final ThreadWorkQueue<QueuedRequest> waitingRequests = new ThreadWorkQueue<>();
         @GuardedBy("this")
         private SimpleLimiter<Void> limiter;
         @GuardedBy("this")
         private ScheduledFuture<?> timeoutCleanup;
         private final Key limiterKey;
         private final Supplier<SimpleLimiter<Void>> limiterFactory;
-
-        private final SafeArg<Optional<String>> safeArgMethod;
```

Comment:
why the change?

Reply:
figure there's still two safeargs which will get made on ever call, figured it's easier to avoid them